test/integration/full/landmark-main-is-top-level/landmark-main-is-top-level-fail.js

```diff
@@ -0,0 +1,38 @@
+
+describe('landmark-main-is-top-level test fail', function () {
+	'use strict';
+	var results;
+	before(function (done) {
+		window.addEventListener('load', function () {
+			axe.run({ runOnly: { type: 'rule', values: ['landmark-main-is-top-level'] } }, function (err, r) {
+				assert.isNull(err);
+				results = r;
+				done();
+			});
+		});
+	});
+
+	describe('violations', function () {
+		it('should find 1', function () {
+			assert.lengthOf(results.violations, 1);
```

Comment:
Add a node count check too:

`assert.lengthOf(results.violations[0].nodes, 2);`

Reply:
@ssanaul This issue isn't addressed yet.

Reply:
I thought 3e5c6ad6d39e6b5de278a2497587225042442431 resolved it. It checks for 4 nodes because I believe there are 4 violations.